lib/puppet/provider/package/version/range/gt.rb

```diff
@@ -0,0 +1,12 @@
+require 'puppet/provider/package/version/range'
```

Comment:
Do I still need to require `range` here, while it is also required in `range/simple`?